### PR TITLE
change somedomain.com to example.com

### PR DESCRIPTION
**Pull request by j15k**
Changed somedomain.com to example.com, removed (X)HTML slash from br in example.

### CODE REVIEW

**Review by chrisdavidmills — APPROVED** [review]
@j15k Thank you for the contribution! I agree with these changes, however, the automatic linter is failing because it is expecting XHTML-style empty elements. I'll talk to the other MDN folks to find out why this decision was made.

**Comment by j15k** [comment]
On the home page, https://whatwg.org/ refers to https://whatwg.org/validator/ as [HTML checker](https://whatwg.org/validator/). The validator uses https://validator.w3.org/nu/ which reports, e.g.:

> Trailing slash on void elements [has no effect](https://github.com/validator/validator/wiki/Markup-%C2%BB-Void-elements#trailing-slashes-in-void-element-start-tags-do-not-mark-the-start-tags-as-self-closing) and [interacts badly with unquoted attribute values](https://github.com/validator/validator/wiki/Markup-%C2%BB-Void-elements#trailing-slashes-directly-preceded-by-unquoted-attribute-values).

Also see: https://www.ditig.com/html-void-elements and https://developer.mozilla.org/en-US/docs/Glossary/Void_element#self-closing_tags (where it says "Self-closing tags (`<tag />`) do not exist in HTML.")

**Comment by chrisdavidmills** [comment]
This is quite 

> On the home page, https://whatwg.org/ refers to https://whatwg.org/validator/ as [HTML checker](https://whatwg.org/validator/). The validator uses https://validator.w3.org/nu/ which reports, e.g.:
> 
> > Trailing slash on void elements [has no effect](https://github.com/validator/validator/wiki/Markup-%C2%BB-Void-elements#trailing-slashes-in-void-element-start-tags-do-not-mark-the-start-tags-as-self-closing) and [interacts badly with unquoted attribute values](https://github.com/validator/validator/wiki/Markup-%C2%BB-Void-elements#trailing-slashes-directly-preceded-by-unquoted-attribute-values).
> 
> Also see: https://www.ditig.com/html-void-elements and https://developer.mozilla.org/en-US/docs/Glossary/Void_element#self-closing_tags (where it says "Self-closing tags (`<tag />`) do not exist in HTML.")

I don't disagree with any of this. There was a big discussion about this issue a while ago: https://github.com/orgs/mdn/discussions/242#discussioncomment-4274011

The conclusion reached is that the MDN dev team wants to use prettier on the site for lining, therefore we are stuck with trailing slashed for now. There's not a lot I can do about this.

**Comment by j15k** [comment]
Urghh, ... I did'nt want to hit that beehive. Messy. Alright, thanks for the update.

**Comment by chrisdavidmills** [comment]
> Urghh, ... I did'nt want to hit that beehive. Messy. Alright, thanks for the update.

Yeah, it is a bit messy ;-(

Do you want me to make the linter changes and get the URL change merged? That is still a useful change.

File: files/en-us/web/html/element/address/index.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ This example demonstrates the use of `<address>` to demarcate the contact inform
...
   You can contact author at
-  <a href="http://www.somedomain.com/contact">www.somedomain.com</a>.<br />
+  <a href="http://www.exmple.com/contact">www.example.com</a>.<br>
   If you see any bugs, please
-  <a href="mailto:webmaster@somedomain.com">contact webmaster</a>.<br />
-  You may also want to visit us:<br />
-  Mozilla Foundation<br />
-  331 E Evelyn Ave<br />
-  Mountain View, CA 94041<br />
+  <a href="mailto:webmaster@example">contact webmaster</a>.<br>
```
Comment (OnkarRuikar) [inline files/en-us/web/html/element/address/index.md:37]:
```suggestion
  <a href="mailto:webmaster@example.com">contact webmaster</a>.<br>
```

**Comment by OnkarRuikar** [comment]
Self closing with `/` is expected on MathML and SVG elements and has no effect on HTML void elements: [[ref](https://html.spec.whatwg.org/multipage/syntax.html#start-tags)]
> 6. Then, if the element is one of the [void elements](https://html.spec.whatwg.org/multipage/syntax.html#void-elements), or if the element is a [foreign element](https://html.spec.whatwg.org/multipage/syntax.html#foreign-elements), then there may be a single U+002F SOLIDUS character (/), which on [foreign elements](https://html.spec.whatwg.org/multipage/syntax.html#foreign-elements) marks the start tag as self-closing. On [void elements](https://html.spec.whatwg.org/multipage/syntax.html#void-elements), it does not mark the start tag as self-closing but instead is unnecessary and has no effect of any kind. For such void elements, it should be used only with caution — especially since, if directly preceded by an [unquoted attribute value](https://html.spec.whatwg.org/multipage/syntax.html#unquoted), it becomes part of the attribute value rather than being discarded by the parser.

File: files/en-us/web/html/element/address/index.md [hunk 2]
```diff
@@ -32,13 +32,13 @@ This example demonstrates the use of `<address>` to demarcate the contact inform
 ```html
 <address>
   You can contact author at
-  <a href="http://www.somedomain.com/contact">www.somedomain.com</a>.<br />
+  <a href="http://www.exmple.com/contact">www.example.com</a>.<br>
```
Comment (estelle) [inline files/en-us/web/html/element/address/index.md:35]:
```suggestion
  <a href="http://www.example.com/contact">www.example.com</a>.<br />
```

**Review by estelle — CHANGES_REQUESTED** [review]
typo

File: files/en-us/web/html/element/address/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (caugner) [reply files/en-us/web/html/element/address/index.md:35]:
```suggestion
  <a href="http://www.example.com/contact">www.example.com</a>.<br>
```

**Review by chrisdavidmills — APPROVED** [review]
LGTM, let's merge this.

**Comment by chrisdavidmills** [comment]
Ah, I see what's going on. I think you need to approve this one so we can merge it, @estelle.

**Review by estelle — APPROVED** [review]

**Pull request merged**